src/components/Dropdown/asyncDropdown/AsyncSearchTags.tsx

```diff
@@ -95,7 +98,11 @@ const AsyncSearchTags = ({
         labelField={'title'}
         values={props.selectedItems}
         testid="multiselect"
+        value={props.value}
         removeItem={props.removeItem}
+        onBlur={props.onBlur}
+        onChange={props.onChange}
+        onKeyDown={onKeyDown}
```

Comment:
La på ekstra props for å kompensere for endringa her: https://github.com/NDLANO/editorial-frontend/commit/1ec8498d334b2c0381d1ac205ed34c6b2249cf59#diff-fd9b0b338f7d02511b8089f52d2768e168e2715f160260329bedf2b963d47937